reactNative/EthereumProject/TransactionsActions.js

```diff
@@ -0,0 +1,6 @@
+export const addTransactions = transactionsIndex => (
```

Comment:
the `actions` and `reducers` should be moved into their own files. An example would be to have a `data` folder which would have a structure like
```
- data
  - transactions
    - actions.js
    - reducer.js
```